src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -333,11 +333,30 @@ def copy_to_start_transfer(filepath='', type='', accession='', transfer_metadata
     return destination
 
 
+def _source_transfers_gave_uuids_to_directories(files):
+    """Returns ``True`` if any of the ``Transfer`` models (that any of the
+    ``File`` models referenced in ``files`` were accessioned by) gave UUIDs to
+    directories. If ``True`` is returned, we assign new UUIDs to all
+    directories in the arranged SIP.
+    """
+    file_uuids = filter(None, [file_.get('uuid') for file_ in files])
+    return any([
+        rslt[0] for rslt in
+        models.Transfer.objects.filter(
+            file__uuid__in=file_uuids).values_list(
+                'diruuids').distinct()])
+
+
 def create_arranged_sip(staging_sip_path, files, sip_uuid):
-    shared_dir = django_settings.SHARED_DIRECTORY
+    shared_dir = helpers.get_server_config_value('sharedDirectory')
```

Comment:
I think this change should be deleted, `get_server_config_value` was recently deleted.